benchmark/child_process/child-process-read.js

```diff
@@ -20,7 +20,7 @@ function main(conf) {
   const len = +conf.len;
 
   const msg = '"' + Array(len).join('.') + '"';
-  const options = { 'stdio': ['ignore', 'pipe', 'ignore'] };
+  const options = {'stdio': ['ignore', 'ipc', 'ignore']};
```

Comment:
Merge error on my part that undoes a change that came in after the commit I reverted. Will fix it momentarily...